src/azul/service/responseobjects/buffer.py

```diff
@@ -0,0 +1,55 @@
+from logging import getLogger
+from io import BytesIO
+from typing import Callable
+
+logger = getLogger(__name__)
+
+
+class FlushableBuffer(BytesIO):
+    """
+    A buffer that flushes the output to a callback function (``inbetween_callback``),
+    either if there exists the size of the fragment is large enough
+    (over ``chunk_size``) on flush or when the buffer is closed.
+
+    :param chunk_size: The minimum size of buffer (byte length)
```

Comment:
I thought we agreed for chunk_size to be the exact size, not the minimum size of each chunk (except the last chunk).

Reply:
Sorry, I missed that.